sprint-planning/readme.md

```diff
@@ -1,29 +1,83 @@
 # Planning
 
-## Content TODO's
+## Goals
 
-- Complete outline below
-- Determining team capacity
-- Tasking sprint stories
-  - size of tasks
-- WIP limits / dividing up into workstreams
+During the [sprint planning](https://www.agilealliance.org/glossary/sprint-planning), the team discusses and agrees on the scope for the upcoming sprint.
 
-## Goals
+**Goals are:**

```

Comment:
Nit: Goals are: -> Goals:

Suggestion - can we avoid mixing **bold** and non bold list titles in the document, I think this would be as readable without the bold, but if you feel the bold is necessary I would suggest also making General guidance bold.

Reply:
Good suggestion. Removed it in upcoming update